app/helpers/admin/corporate_information_pages_helper.rb

```diff
@@ -0,0 +1,48 @@
+module Admin::CorporateInformationPagesHelper
+  include Admin::EditionsHelper
+
+  def index_table_title_row(edition)
+    title = edition.title
+    title += " (#{edition.primary_locale})" if edition.non_english_edition?
+    sanitize(title)
+  end
+
+  def index_table_state_row(edition)
+    state = if edition.force_published?
+              "Force published"
+            elsif edition.unpublishing.present?
+              "Unpublished"
+            else
+              admin_edition_state_text(edition)
+            end
+
+    tags = case state
+           when "Force published"
+             tag.span(state, class: "govuk-tag govuk-tag--s govuk-tag--yellow")
+           when "Draft"
+             tag.span(state, class: "govuk-tag govuk-tag--s govuk-tag--blue")
+           when "Submitted"
+             tag.span(state, class: "govuk-tag govuk-tag--s")
+           when "Published"
+             tag.span(state, class: "govuk-tag govuk-tag--s govuk-tag--green")
+           when "Scheduled"
+             tag.span(state, class: "govuk-tag govuk-tag--s govuk-tag--turquoise")
+           when "Rejected"
+             tag.span(state, class: "govuk-tag govuk-tag--s govuk-tag--red")
+           when "Withdrawn", "Unpublished"
+             tag.span(state, class: "govuk-tag govuk-tag--s govuk-tag--grey")
+           end
```

Comment:
At some point soon, I expect these tags will be needed in more places than just the Corporate Information Pages index. I'd hope for the sake of consistency that we end up re-using it on the Documents index page, and other sections that need to list Editions.

So if not now, I think we'll _very soon_ need to consider moving this into the Editions helper rather than a specific CorporateInformationPages helper.

Reply:
Yea good shout. I'll move it to the editions helper.